include/samplers/ellipsoid_walks/dikin_walker.h

```diff
@@ -0,0 +1,93 @@
+// Code from https://github.com/rzrsk/vaidya-walk
```

Comment:
You can add
`// Related publication: Chen et al. - "Fast MCMC Sampling Algorithms on Polytopes", Journal of Machine Learning Research 19 (2018) 1-86.`
similarly to all files

Also please add the info and copyright notes as in all the files of the library